scripts/scripts.js

```diff
@@ -850,13 +850,13 @@ function unwrapBlock(block) {
       appendTo = postBlockSection;
     }
   });
-  if (!section.hasChildNodes()) {
+  if (section.childElementCount === 0) {
     section.remove();
   }
-  if (!blockSection.hasChildNodes()) {
+  if (blockSection.childElementCount === 0) {
     blockSection.remove();
   }
-  if (!postBlockSection.hasChildNodes()) {
+  if (postBlockSection.childElementCount === 0) {
     postBlockSection.remove();
   }
```

Comment:
@kptdobe , I see other occurrences of `hasChildNode` like here https://github.com/adobe/blog/blob/9675eadc2037547ecd0460e0f18b23cb1fb6079e/blocks/footer/footer.js#L51-L60 . Should these also be changed similarly?

Reply:
Good catch. Checking...